templates/octobox/index.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Output,
+  randomPassword,
+  Services
+} from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const databasePassword = randomPassword();
+  const redisPassword = randomPassword();
+
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      env: [
+        `RAILS_ENV=production`,
```

Comment:
This either needs to be changed to `development` or the `SECRET_KEY_BASE` env needs to be added.